packages/common/src/types/visualizations.ts

```diff
@@ -0,0 +1,31 @@
+export type BarChartConfig = {
+    metadata: {
+        version: number;
+    };
+    // TODO: There was some back and forth about whether this should be 'defaultType'
+    // I'm not sure
+    type: 'barChart';
+    style: {
+        legend:
+            | {
+                  position: 'top' | 'bottom' | 'left' | 'right';
+                  align: 'start' | 'center' | 'end';
+              }
+            | undefined;
+    };
+    axesConfig: {
+        x: {
+            reference: string;
+            label: string;
+        };
+        y: {
+            reference: string;
```

Comment:
we don't need `reference` here - we should map from the `seriesConfig`

Reply:
I don't follow. These should be the axis configurations for the various series. So not just things like label, but also it could be things like tick spacing. I think we'll need it.

Reply:
Sharing an example soon!